### ISSUE TITLE
Release date of Autocomplete-light v4?

### ISSUE THREAD

**Issue by MErenB** [yourlabs/django-autocomplete-light#1285]
Greetings Yourlabs team,
I am working on a project that is being ported from Django 1.8 to Django 4.0. Since latest django-autocomplete-light release (v3.9) is not supports Django 4.0, we cannot use django-autocomplete-light. 
We're wondering when the django-autocomplete-light v4 will be released. Should we wait for its release, or should we find a replacement for our autocompletion needs?
If a replacement is needed, have you got any recommendations about which package to use?

**Comment by jpic** [comment]
You can use django-autocomplete-light==3.9.0rc6 meanwhile

**Comment by jpic** [comment]
For alternatives, see here: https://djangopackages.org/grids/g/auto-complete/

**Comment by jpic** [comment]
Sorry I'm a bit confused with your questions:

- Support for Django 4: already released on Feb 3rd in 3.9.1 already
- DAL v4: no date planned, are you expecting a specific feature from DAL v4? Currently, only FK support is tested

Please reopen if another answer is needed

**Comment by mcanu** [comment]
I also got confused by DAL not supporting Django 4 yet because of the [README's Roadmap section](https://github.com/yourlabs/django-autocomplete-light#roadmap).

> - DAL < 4 supports Python 2 and 3 and Django 1.8 to 3.x
> - DAL 4 supports Django 3.x and 4.x and Python 3
> - DAL >= 3.9 offers a modern alternative to select2

**Comment by gdvalderrama** [comment]
Yeah, the READ.me is also misleading, referring to Django LTS as 1.11

> Python 2.7, 3.4, Django 2.0+ support (Django 1.11 (LTS), is supported until django-autocomplete-light-3.2.10),

**Comment by jpic** [comment]
Yep, sorry this repo is 11 years old today :birthday: 

Anyone wants to update this?
